README.md

```diff
@@ -182,6 +182,9 @@ Since Sony likes to ruin a perfect thing:
 While Microsoft doesn't provide an official unlocking method, an [unofficial method](https://github.com/WOA-Project/SurfaceDuo-Guides/blob/main/Install/UnlockingBootloader.md) has been available for their Android devices basically ever since Microsoft began making Android phones, and Microsoft hasn't bothered to patch it, and Windows is possible to run on an unlocked bootloader Surface Duo, so it seems like the bootloader will remain unlockable for now, but do note Microsoft can patch the unofficial method at any time.<br/>
 As for Windows Phones, once again, no official methods, but an [unofficial method](http://allaboutwindowsphone.com/features/item/24245_Aguideforunlockingthebootloade.php) has been around for a while, and supports all Mirosoft-made Lumias except for the 540 and 535, as Windows Phone hasn't been supported since 2017, it is highly unlikely this is going to be patched.
 
+### Umidigi
+Umidigi phones are generally unlockable, however due to them being a budget-focused manufacturer, they use Unisoc SoCs, and [Unisoc requires a bunch of extra steps to unlock for whatever reason](https://www.hovatek.com/forum/thread-32287.html), and therefore I wouldn't consider them completely safe. MediaTek Umidigi devies can just use flashing unlock/oem unlock in fastboot.
```

Comment:
If the process doesn't require an internet connection, it's fine to put it in "safe for now", just at the bottom.

I also don't really like the hovatek link, as their forum seems to require registration to download.

Reply:
I've moved it to safe for now, hovatek was sadly the best source I could find.